utils/utils.go

```diff
@@ -54,3 +55,45 @@ func StringInSlice(input []string, str string) bool {
 	exists, _ := Contains(result, str)
 	return exists
 }
+
+// SliceTrimSpace delete empty item, like " ", "\t", "\n" from slice
+func SliceTrimSpace(input []string) []string {
+	output := []string{}
+	for _, item := range input {
+		str := strings.TrimSpace(item)
+		if str != "" {
+			output = append(output, str)
+		}
+	}
+
+	return output
+}
+
+// StringSliceEqual compare two string slice, ignore the order.
+func StringSliceEqual(s1, s2 []string) bool {
```

Comment:
`[]string{"a", "a", "b"}` is not equal to `[]string{"a", "b","b"}`. but `StringSliceEqual` return true

Reply:
Great catch